### PR TITLE
i18n: Italian

### PR DESCRIPTION
**Pull request by UserRoot-Luca**
Italian translation

### CODE REVIEW

File: app/i18n/it/conf.php [hunk 1]
```diff
@@ -268,7 +268,7 @@
 			'older_first' => 'Prima i più vecchi',
 		),
 		'star' => array(
-			'when' => 'Mark an article as favourite…',	// TODO
+			'when' => 'Segna un articolo come preferito...',
```
Comment (Alkarex) [inline app/i18n/it/conf.php]:
```suggestion
			'when' => 'Segna un articolo come preferito…',
```

File: app/i18n/it/sub.php [hunk 2]
```diff
@@ -138,12 +138,12 @@
...
 			),
 			'json_dotnotation' => array(
-				'_' => 'JSON (dot notation)',	// TODO
+				'_' => 'JSON (notazione a punti)',
 				'feed_title' => array(
 					'_' => 'titolo feed',
 					'help' => 'Esempio: <code>meta.title</code> o una stringa statica: <code>"Il mio feed personalizzato"</code>',
 				),
-				'help' => 'A JSON dot notated uses dots between objects and brackets for arrays (e.g. <code>data.items[0].title</code>)',	// TODO
+				'help' => 'Una notazione JSON utilizza i punti tra gli oggetti e le parentesi per gli array (p.es. <code>data.items[0].title</code>)',
```
Comment (Alkarex) [inline app/i18n/it/sub.php]:
```suggestion
				'help' => 'Una notazione JSON utilizza i punti tra gli oggetti e le parentesi per gli array. P.es. <code>data.items[0].title</code>',
```

Could that be acceptable? For technical reasons, `array (` is not supported at the moment inside a sentence

Comment (UserRoot-Luca) [reply app/i18n/it/sub.php]: @Alkarex I just translated I didn't want to change the formatting of the text so I found it so I translated it

File: app/i18n/it/sub.php [hunk 3]
```diff
@@ -229,8 +229,8 @@
 		'title_add' => 'Aggiungi RSS feed',
 		'ttl' => 'Non aggiornare automaticamente piu di',
 		'unicityCriteria' => array(
-			'_' => 'Article unicity criteria',	// TODO
-			'forced' => '<span title="Block the unicity criteria, even when the feed has duplicate articles">forced</span>',	// TODO
+			'_' => 'Criteri di unicità dell’articolo',
+			'forced' => '<span title="Blocca i criteri di unicità, anche quando il feed ha articoli duplicati">forced</span>',	// DIRTY
```
Comment (Alkarex) [inline app/i18n/it/sub.php]:
```suggestion
			'forced' => '<span title="Blocca i criteri di unicità, anche quando il feed ha articoli duplicati">forced</span>',
```

File: app/i18n/it/sub.php [hunk 4]
```diff
@@ -31,7 +31,7 @@
 		'empty' => 'Categoria vuota',
 		'expand' => 'Espandi categoria',
 		'information' => 'Informazioni',
-		'open' => 'Open category',	// TODO
+		'open' => 'Categoria aperta',
```
Comment (Alkarex) [inline app/i18n/it/sub.php]:
```suggestion
		'open' => 'Aprire la categoria',
```

File: app/i18n/it/sub.php [hunk 5]
```diff
@@ -200,7 +200,7 @@
 		),
 		'no_selected' => 'Nessun feed selezionato.',
 		'number_entries' => '%d articoli',
-		'open_feed' => 'Open feed %s',	// TODO
+		'open_feed' => 'Feed aperto %s',
```
Comment (Alkarex) [inline app/i18n/it/sub.php]:
```suggestion
		'open_feed' => 'Aprire il feed %s',
```

File: app/i18n/it/sub.php (hunk 3 above)
Comment (UserRoot-Luca) [reply app/i18n/it/sub.php]: The term `forced` should be translated but I am not sure how to translate it

**Comment by UserRoot-Luca** [comment]
@Alkarex Now all the untranslated text is translated

**Comment by Alkarex** [comment]
Grazie!
Please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/edge/CREDITS.md

**Pull request merged**